armi/reactor/blueprints/isotopicOptions.py

```diff
@@ -393,15 +393,11 @@ def apply(self, material):
         material.massFrac = dict(self.massFracs)
         if self.density is not None:
             if not isinstance(material, materials.Custom):
-                runLog.warning(
-                    "You either specified a custom mass density or number densities "
-                    "(which implies a mass density) on `{}` with custom isotopics `{}`. "
-                    "This has no effect on this Material class; you can only "
-                    "override mass density on `Custom` "
-                    "materials. Consider switching to number fraction input. "
-                    "Continuing to use {} mass density.".format(
-                        material, self.name, material
-                    )
+                runLog.important(
+                    "A custom density or number densities has been specified for a non-custom "
+                    "material. The material density will not be updated to prevent unintentional "
```

Comment:
`material object's density`
or
"The component's density has been updated to reflect the change, but the material it points to preserves the actual material density (just like for 2d expansion)"

Reply:
```suggestion
                    "material. The material object's density will not be updated to prevent unintentional "
```